lib/eslint.js

```diff
@@ -26,7 +26,10 @@ const assert = require("assert"),
     Traverser = require("./util/traverser"),
     RuleContext = require("./rule-context"),
     rules = require("./rules"),
-    timing = require("./timing");
+    timing = require("./timing"),
+
```

Comment:
That was intentional. Following the style of https://github.com/eslint/eslint/blob/655a3cfee6319ac9e7d0f7780eb5dcce78e611b6/lib/cli-engine.js#L33